docs/examples/spam/meson.build

```diff
@@ -3,39 +3,30 @@
 # SPDX-License-Identifier: MIT
 
 project(
-  'spam',
-  'c',
+    'spam',
+    'c',
 )
 
 py_mod = import('python')
 py = py_mod.find_installation()
 
 # Specify directories to be included - usually this is where your header files live.
 # Note that specifying the path to Python.h is not necessary with meson-python.
-includes = include_directories(
-  [
-    'src'
-  ]
-)
+includes = include_directories(['src'])
 
 srcs = [
-  'src/spammodule.c',
+    'src/spammodule.c',
 ]
 
 # If some files (such as .py files) need to be copied to site-packages,
 # this is where they get specified. Files get copied to
 # <python directory>/site-packages/<subdir>
-py.install_sources(
-  [
-    'src/__init__.py'
-  ],
-  subdir: 'spam'
-)
+py.install_sources(['src/__init__.py'], subdir: 'spam')
```

Comment:
I don't like how muon-fmt likes to format the code. More about this later, but can it at least be made to be consistent with itself? For example, here it prefers to have a list of one element with the delimiters inline and additional arguments to follow...